rtrlib/rtr_mgr.h

```diff
@@ -67,28 +68,37 @@ struct rtr_mgr_group {
 	enum rtr_mgr_status status;
 };
 
+//TODO Find a nicer way todo a linked list (without writing our own)
+struct rtr_mgr_group_node {
```

Comment:
We need a way to access the the group list tough or at least access the active group, otherwise it would not be possible to get status information beyond "it still works somehow".

Of course that can also be archived with helper functions.